### PR TITLE
update absolute links in learning path

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: _index.md [hunk 1]
```diff
@@ -9,131 +9,131 @@ hide_commit: true
...
 
-[TiUniManager](https://docs.pingcap.com/zh/tidb/dev/tiunimanager-overview)
+[TiUniManager](https://docs.pingcap.com/zh/tidb/v7.5/tiunimanager-overview)
 
 </LearningPath>
 
 <LearningPath label="参考" icon="cloud-dev">
 
-[TiDB 路线图](https://docs.pingcap.com/zh/tidb/dev/tidb-roadmap)
+[TiDB 路线图](https://docs.pingcap.com/zh/tidb/v7.5/tidb-roadmap)
```
Comment (qiancai) [inline _index.md:124]:
```suggestion
[TiDB 路线图](https://docs.pingcap.com/zh/tidb/dev/tidb-roadmap)
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: _index.md [hunk 2]
```diff
@@ -9,131 +9,131 @@ hide_commit: true
...
 <LearningPath label="开发" icon="doc8">
 
-[开发者手册概览](https://docs.pingcap.com/zh/tidb/dev/dev-guide-overview)
+[开发者手册概览](https://docs.pingcap.com/zh/tidb/v7.5/dev-guide-overview)
 
-[快速开始](https://docs.pingcap.com/zh/tidb/dev/dev-guide-build-cluster-in-cloud)
+[快速开始](https://docs.pingcap.com/zh/tidb/v7.5/dev-guide-build-cluster-in-cloud)
 
-[示例程序](https://docs.pingcap.com/zh/tidb/dev/dev-guide-sample-application-spring-boot)
+[示例程序](https://docs.pingcap.com/zh/tidb/v7.5/dev-guide-sample-application-spring-boot)
```
Comment (Oreoxmt) [inline _index.md:36]:
```suggestion
[示例程序](https://docs.pingcap.com/zh/tidb/v7.5/dev-guide-sample-application-java-spring-boot)
```

**Pull request merged**